src/FixedDataTableBufferedRows.js

```diff
@@ -101,7 +101,7 @@ var FixedDataTableBufferedRows = createReactClass({
   },
 
   _updateBuffer() {
-    if (this.isMounted()) {
+    if (this._rowBuffer) {
```

Comment:
Hmm it seems fine to me, but ya removing isMounted wherever possible sounds good.